packages/drivers/odsp-driver/src/graph.ts

```diff
@@ -109,12 +110,19 @@ export async function graphFetch(
             siteUrl,
         );
     const url = graphUrl.startsWith("http") ? graphUrl : `https://graph.microsoft.com/v1.0/${graphUrl}`;
+    const retryFilter: RetryFilter<Response> = allowlist([offlineFetchFailureStatusCode, fetchFailureStatusCode]);
```

Comment:
Why only these two codes? Can we use the logic of createOdspNetworkError() to determine what is retryable and what is not, such that we can benefit from future changes in that function?

Reply:
What about 429 and respecting server request to come back in a minute? Lack of that could cause some damage when server has trouble.

I'd rather see us getting rid of fetchWithRetry() altogether and seeing how we can generalize RetriableDocumentStorageService.readWithRetry to be useful outside the class it's in right now.

Reply:
I'd probably leave replacing fetchWithRetry for later (as it's involved change), but I'd try to use createOdspNetworkError here for translation, and usage of retryAfter for 429 management

Reply:
Yes I was thinking about that but added this as a quick fix.

Reply:
How hard is it to start using createOdspNetworkError flow in here? I feel like it should not be that hard.
Leave it up to you to decide, but let's definitely open an issue to follow up

Reply:
Replaced in this change only.